presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -1447,6 +1448,38 @@ public void testMaterializedViewOptimizationWithDerivedFieldsWithAlias()
         }
     }
 
+    // End to end test for base query optimization rewrite using materialized view
+    @Test(enabled = false)
```

Comment:
@gggrace14 PR for refresh has been merged. You should be able to enable these test cases now.

Reply:
@jainxrohit This test case won't work since it still relies on fetching the base-to-mv mapping (currently it is fetched/hardcoded in the optimization rewriter utils). Once the metastore is updated with the base-ti-mv information, we would be able to enable this testcase.